draft-irtf-cfrg-opaque.md

```diff
@@ -2275,7 +2292,9 @@ outputs computed during the authentication of an unknown or unregistered user. N
 All values are encoded in hexadecimal strings. The configuration information
 includes the (OPRF, Hash, KSF, KDF, MAC, Group, Context) tuple, where the Group
 matches that which is used in the OPRF. These test vectors were generated using
-draft-10 of {{OPRF}}.
+draft-21 of {{OPRF}}. The KSF used for each test vector is the identity function
```

Comment:
May I suggest to use argon2id but with short `m`, for example instead `m = 2^21` set `m=8`.